test/e2e/e2e_test.go

```diff
@@ -238,7 +240,7 @@ func (o *options) DefaultClusterOptions() core.CreateOptions {
 			ServicePublishingStrategy: kubevirt.IngressServicePublishingStrategy,
 			ContainerDiskImage:        o.configurableClusterOptions.KubeVirtContainerDiskImage,
 			Cores:                     2,
-			Memory:                    "4Gi",
+			Memory:                    o.configurableClusterOptions.KubeVirtNodeMemory,
```

Comment:
Why make this configurable rather than just hardcoding a working size? This now requires users to know what the right size it

Reply:
There's still the cli default of "4Gi".

I was hoping to make the memory size configurable because it allows us to adapt our prow configs without needing to do this dance between modifying the hypershift codebase and then modifying prow.

For example, today the e2e tests use 5 replicas of 4Gi worker nodes. This configuration is occasionally failing because in 4.11 we have some pods that fail to schedule on the workers due to 4Gi being too little memory per worker.

I think 2 replica's of 8Gi will more reliably fit all the pods. I'd like to make that kind of change just in the prow config instead of having to change a hardcoded value in hypershift's e2e test which will cause the kubevirt e2e tests to fail until i can update the prow config to modify the replica count.

basically, memory is closely related to replica count for us, so it would be helpful to expose this via the cli just like replica count.